2023-11-24 17:02:18,984 INFO    MainThread:25289 [wandb_init.py:_log_setup():292] Logging user logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231124_170218-2w6ci1uv/logs/debug.log
2023-11-24 17:02:18,985 INFO    MainThread:25289 [wandb_init.py:_log_setup():293] Logging internal logs to /content/drive/MyDrive/on-policy/onpolicy/scripts/results/StarCraft2/2m_vs_1z/rmappo/check/wandb/run-20231124_170218-2w6ci1uv/logs/debug-internal.log
2023-11-24 17:02:18,985 INFO    MainThread:25289 [wandb_setup.py:_flush():68] setting env: {}
2023-11-24 17:02:18,985 INFO    MainThread:25289 [wandb_setup.py:_flush():68] setting user settings: {}
2023-11-24 17:02:18,985 INFO    MainThread:25289 [wandb_setup.py:_flush():68] multiprocessing start_methods=fork,spawn,forkserver
2023-11-24 17:02:20,887 INFO    MainThread:25289 [wandb_run.py:_console_start():1265] atexit reg
2023-11-24 17:02:20,898 INFO    MainThread:25289 [wandb_run.py:_redirect():1135] redirect: SettingsConsole.REDIRECT
2023-11-24 17:02:20,900 INFO    MainThread:25289 [wandb_run.py:_redirect():1138] Redirecting console.
2023-11-24 17:02:20,900 INFO    MainThread:25289 [redirect.py:install():196] install start
2023-11-24 17:02:20,901 INFO    MainThread:25289 [redirect.py:install():211] install stop
2023-11-24 17:02:20,902 INFO    MainThread:25289 [redirect.py:install():196] install start
2023-11-24 17:02:20,904 INFO    MainThread:25289 [redirect.py:install():211] install stop
2023-11-24 17:02:20,910 INFO    MainThread:25289 [wandb_run.py:_redirect():1182] Redirects installed.
2023-11-24 17:02:22,216 INFO    Dummy-4   :25289 [wandb_run.py:_atexit_cleanup():1238] got exitcode: 1
2023-11-24 17:02:22,218 INFO    Dummy-4   :25289 [wandb_run.py:_restore():1210] restore
2023-11-24 17:02:22,218 INFO    Dummy-4   :25289 [redirect.py:uninstall():215] uninstall start
2023-11-24 17:02:22,219 INFO    Dummy-4   :25289 [redirect.py:_stop():264] _stop: stdout
2023-11-24 17:02:22,220 INFO    stdout    :25289 [redirect.py:_pipe_relay():119] relay done saw last write: stdout
2023-11-24 17:02:22,220 INFO    stdout    :25289 [redirect.py:_pipe_relay():135] relay done done: stdout
2023-11-24 17:02:22,220 INFO    Dummy-4   :25289 [redirect.py:_stop():270] _stop closed: stdout
2023-11-24 17:02:22,220 INFO    Dummy-4   :25289 [redirect.py:_stop():276] _stop joined: stdout
2023-11-24 17:02:22,221 INFO    Dummy-4   :25289 [redirect.py:_stop():278] _stop rd closed: stdout
2023-11-24 17:02:22,221 INFO    Dummy-4   :25289 [redirect.py:uninstall():219] uninstall done
2023-11-24 17:02:22,221 INFO    Dummy-4   :25289 [redirect.py:uninstall():215] uninstall start
2023-11-24 17:02:22,221 INFO    Dummy-4   :25289 [redirect.py:_stop():264] _stop: stderr
2023-11-24 17:02:22,221 INFO    stderr    :25289 [redirect.py:_pipe_relay():119] relay done saw last write: stderr
2023-11-24 17:02:22,222 INFO    stderr    :25289 [redirect.py:_pipe_relay():135] relay done done: stderr
2023-11-24 17:02:22,221 INFO    Dummy-4   :25289 [redirect.py:_stop():270] _stop closed: stderr
2023-11-24 17:02:22,222 INFO    Dummy-4   :25289 [redirect.py:_stop():276] _stop joined: stderr
2023-11-24 17:02:22,222 INFO    Dummy-4   :25289 [redirect.py:_stop():278] _stop rd closed: stderr
2023-11-24 17:02:22,222 INFO    Dummy-4   :25289 [redirect.py:uninstall():219] uninstall done
2023-11-24 17:02:22,229 INFO    Dummy-4   :25289 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "8194bfba2d32423db9e7e629e6896218"
response {
  poll_exit_response {
    file_counts {
      wandb_count: 1
    }
    pusher_stats {
      uploaded_bytes: 1311
      total_bytes: 1311
    }
  }
}

2023-11-24 17:02:24,235 INFO    Dummy-4   :25289 [wandb_run.py:_wait_for_finish():1349] got exit ret: uuid: "40a3e67d2f6748cfa8e5427eac27e0ef"
response {
  poll_exit_response {
    done: true
    exit_result {
    }
    file_counts {
      wandb_count: 5
    }
    pusher_stats {
      uploaded_bytes: 10659
      total_bytes: 10659
    }
  }
}

2023-11-24 17:02:25,345 INFO    Dummy-4   :25289 [wandb_run.py:_show_files():1543] logging synced files
2023-11-24 17:02:25,346 INFO    Dummy-4   :25289 [internal.py:handle_exit():137] Internal process exited
